docker-compose-generator/docker-fragments/bitcoin.yml

```diff
@@ -11,7 +11,6 @@ services:
           rpcport=43782
           rpcbind=0.0.0.0:43782
           port=39388
-          disablewallet=1
```

Comment:
Ok so this one is complicated... Old deployment of BTCPay had no `disablewallet`, so a wallet file got created.
Now if you remove this, Bitcoin core will want to rescan from the wallet that got created long ago to the tip. But if we run in pruned money then... the node won't start.

Instead, can you just put the bitcoin core wallet on a separate volume? Not only this prevent this problem but also it will make sure people don't delete their wallet by mistake when they only want to rescan their node.

Reply:
Is this even possible? The wallet is in a directory that is already used as a volume

Reply:
you need to use `walletdir`. But actually the tricky part is that you need a different walletdir depending on the network...
So I think in the end you need to modify the bitcoin core image, so in the entrypoint, you create the folder for the user at the `WALLET_DIR/{network}` environment variable location. Then use walletdir= `WALLET_DIR/{network}`

Reply:
(ノ ゜Д゜)ノ ︵ ┻━┻

Ok will get on it

Reply:
https://github.com/NicolasDorier/docker-bitcoin/pull/21